ansible/kibana.yml

```diff
@@ -7,3 +7,5 @@
 
   roles:
     - role: gsa.ansible-role-kibana
+      vars:
+        kibana_version: "6.6"
```

Comment:
Should this kind of variable be in the playbook or in the inventory?

Reply:
I moved all other vars to inventory, but I wanted to make this declarative in the playbook to make it easier to upgrade and make it more obvious for the user deploying the role.